pkg/provider/acceptance_table_test.go

```diff
@@ -0,0 +1,133 @@
+package provider
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/MaterializeInc/terraform-provider-materialize/pkg/materialize"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/jmoiron/sqlx"
+)
+
+func TestAccTable_basic(t *testing.T) {
+	tableName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTableResource(tableName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTableExists("materialize_table.test"),
+					resource.TestCheckResourceAttr("materialize_table.test", "name", tableName),
```

Comment:
There are a few other attributes we should track like the `schema_name`, `database_name`, `qualified_sql_name`, `column`